plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPageModel.java

```diff
@@ -348,9 +348,12 @@ public IStatus connect() {
 		IStatus status = Status.OK_STATUS;
 		listener.secureStoreException = null;
 		try {
-			IConnection connection = createConnection(connectionFactory, connectionAuthenticationProvider);
+			IConnection connection = 
```

Comment:
this change actually breaks authentication. If you have an existing connection without stored password, it won't let you authenticate. Reverting the changes works